crm/integrations/plivo/handler.py

```
@@ -0,0 +1,227 @@
+    if not from_number:
+        from_number = frappe.get_value("CRM Telephony Agent", {"user": frappe.session.user}, "mobile_no")
+
+    if not caller_id:
+        caller_id = frappe.get_value("CRM Telephony Agent", {"user": frappe.session.user}, "plivo_number")
+
+    if not from_number:
+        frappe.throw(
+            _("You do not have mobile number set in your Telephony Agent"), title=_("Mobile Number Missing")
+        )
+
+    record_call = frappe.db.get_single_value("CRM Plivo Settings", "record_call")
```

Ruff (0.11.9)

Local variable `record_call` is assigned to but never used

Remove assignment to unused variable `record_call`

(F841)